docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -1284,6 +1287,13 @@ contain certificates, CRLs, and much more. PKCS7 files commonly have a ``p7b``,
         :param data: The data to be encrypted.
         :type data: :term:`bytes-like`
 
+    .. method:: set_algorithm(algorithm)
+
+        Sets the content encryption algorithm to use. Only AES-128 and AES-256 are supported.
+        
+        :type algorithm: A :class:`~cryptography.hazmat.primitives.ciphers.algorithms.AES`
+            with a key size of 128 or 256 bits.
```

Comment:
I find this documentation a bit difficult to read.

Reply:
Tried to simplify it, please let me know!